### ISSUE TITLE
Fresh 1.0

### ISSUE THREAD

**Issue by lucacasonato** [denoland/fresh#219]
I am planning to cut a 1.0 release of `fresh` soon. There are a couple of things I need to tackle before this can happen though:

- [x] #123
- [x] #177
- [x] Finish up the "concepts" pages in docs
- [x] Make `preact` version customizable via the import map
- [x] Publish fresh at versioned URLs
- [x] Move the `fresh` repo to the https://github.com/denoland organization
- [x] Fix multiple islands on a page (#196)
- [ ] ~~Support or error on nested islands (#168)~~ out of scope for now

There are a couple of features that I don't consider blockers, but would be very nice to have:
 
- [ ] Ensure `deno vendor` works well, and document this
- [x] Automatic manifest updating during development (blocked on https://github.com/denoland/deno/issues/14685), also see #205

Before cutting the 1.0 release, I will cut a release candidate (1.0.0-rc.0) that I'll test in all of our production projects for a week or so (and I hope you try it out too!).

**Comment by lucacasonato** [comment]
Fresh 1.0.0 has been released: https://github.com/denoland/fresh/releases/tag/1.0.0
